jattoabdul/narwhalorders
# Implement Fetching of "narwhal" orders from shopify into admin

**What does this PR do?**
- [x] It embeds the freshly created app into Shopify admin.
- [x] It adds narwhals order table to DB.
- [x] It adds webhook for order creation and paid events
- [x] It adds a job to pass data from webhook on order creation and paid events
- [x] It store newly-created orders in that have "narwhal" in its line_items
- [ ] Write test

**Description of Task to be completed?**
>- After the app is installed, the app should monitor for new orders. When a new order is received it should check to see if there are any products in the order that contains the word “Narwhal”. If there are, then it should add a record in the DB or somewhere. We’ll call these “Narwhal Orders”.

**How should this be manually tested?**
- Go to the app URL,
- Install the app
- Create a new order or Pay for an order
- Load the app in your Shopify admin and see the newly-created or paid-for order.

**Any background context you want to provide?**
- N/A

**What are the relevant issues?**
- N/A

**Screenshots (if appropriate)**
- 
<img width="1435" alt="Screenshot 2020-06-09 at 5 34 02 PM" src="https://user-images.githubusercontent.com/12856872/84175137-717bf280-aa77-11ea-87d9-28b0d1c8438b.png">


**Questions:**
- N/A
